tests/integration/suite/daprd/conversation/alpha2/grpc/basic.go

```diff
@@ -179,7 +179,7 @@ func (b *basic) Run(t *testing.T, ctx context.Context) {
 		})
 		require.NoError(t, err)
 		// Echo component returns one output per input message
```

Comment:
this comment needs updating if you are changing the behavior on echo component pls